clang/docs/SanitizerSpecialCaseList.rst

```diff
@@ -103,6 +101,22 @@ supported sanitizers.
     char c = toobig; // also not instrumented
   }
 
+Conflicting entries are resolved by the latest entry, which takes precedence.
```

Comment:
If multiple entries match the source, than the latest entry takes the precedence.